tests/dummy/config/ember-cli-update.json

```diff
@@ -0,0 +1,18 @@
+{
+  "schemaVersion": "1.0.0",
+  "packages": [
+    {
+      "name": "ember-cli",
+      "version": "3.21.2",
+      "blueprints": [
+        {
+          "name": "addon",
+          "outputRepo": "https://github.com/ember-cli/ember-addon-output",
+          "codemodsSource": "ember-addon-codemods-manifest@1",
+          "isBaseBlueprint": true,
+          "options": []
```

Comment:
I think if you add `--yarn` here to these `options` `ember-cli-update` will be able to generate the right content for `CONTRIBUTING.md`.

Reply:
@rwjblue didn't know about that, thanks for pointing out!